dora-metrics/pelorus
# fix: Release steps

## Describe the behavior changes introduced in this PR

Add PR template for Pelorus release, which contains steps for completing it. [About PR templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

## Linked Issues

resolves #935 
depends on #942 

## Testing Instructions

- Check if all channels/emails were added to checkboxes
- Check if all steps were added
